module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderText.kt

```diff
@@ -36,3 +86,19 @@ internal fun RenderTextNode(text: Text, modifier: Modifier = Modifier) {
         modifier = modifier
     )
 }
+
+@Composable
+private fun RowScope.RenderTextImage(resource: Resource, imageSize: Int, startPadding: Int = 0, endPadding: Int = 0) {
```

Comment:
instead of sending in `startPadding` and `endPadding`, you should send in a `Modifier`